#!/usr/bin/env python
PACKAGE = "cob_tracking_action"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("timer_value",   int_t,   0, "Timer seconds to wait",  10, 0, 120)
gen.add("base_active",   bool_t,   0, "Command base actively",  False)
gen.add("base_ratio",    double_t, 0, "Value for base_ratio",  0.0, 0, 1)
gen.add("distance_threshold", double_t, 0, "Noise suppression threshold", 0.005, 0.0, 0.5)


exit(gen.generate(PACKAGE, "cob_tracking_action", "TrackingAction"))
